Improve `MeshDataTool.get_face_vertex()` method description

mateuseap opened the pull request:
<!--
Please target the `master` branch in priority.
PRs can target other branches (e.g. `3.2`, `3.5`) if the same change was done in `master`, or is not relevant there.
PRs must not target `stable`, as that branch is updated manually.

The type of content accepted into the documentation is explained here:
https://docs.godotengine.org/en/latest/community/contributing/content_guidelines.html
-->

## What I did 

- Improved the `MeshDataTool.get_face_vertex()` method description by making it more clear about what is the data type that is returned and adding a code example.

#### Closes: [#7852](https://github.com/godotengine/godot-docs/issues/7852) (issue from godot-docs repo)

AThousandShips commented:
Would need a c# example as well, @raulsntos?

raulsntos commented on doc/classes/MeshDataTool.xml:
```diff
@@ -137,8 +137,16 @@
 				Vertex argument must be either 0, 1, or 2 because faces contain three vertices.
+				[b]Example:[/b]
+				[codeblocks]
+				[gdscript]
+				var index = mesh_data_tool.get_face_vertex(0, 1) # Gets the index of the second vertex of the first face.
+				var position = mesh_data_tool.get_vertex(index)
+				var normal = mesh_data_tool.get_vertex_normal(index)
+				[/gdscript]
```
Example translated to C#:

```suggestion
				[gdscript]
				var index = mesh_data_tool.get_face_vertex(0, 1) # Gets the index of the second vertex of the first face.
				var position = mesh_data_tool.get_vertex(index)
				var normal = mesh_data_tool.get_vertex_normal(index)
				[/gdscript]
				[csharp]
				int index = meshDataTool.GetFaceVertex(0, 1); // Gets the index of the second vertex of the first face.
				Vector3 position = meshDataTool.GetVertex(index);
				Vector3 normal = meshDataTool.GetVertexNormal(index);
				[/csharp]
```

clayjohn commented:
Looks great! The final step before merging is to squash all the commits together so that the whole PR only contains 1 big commit with all your changes. We like to merge one commit at a time to keep the git history clean and navigable. 

If you don't know how to do that, we have a helpful tutorial in the official documentation https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#the-interactive-rebase

mateuseap committed:
Improve MeshDataTool.get_face_vertex() method description

Improve MeshDataTool.get_face_vertex() method description and also add an example code.

Co-authored-by: Raul Santos <raulsntos@gmail.com>

akien-mga commented:
I would also suggest rewording the commit message as we don't use this format usually. The title of this PR is a good match for our usual commit style (you can see other examples in `git log`).

mateuseap commented:
> I would also suggest rewording the commit message as we don't use this format usually. The title of this PR is a good match for our usual commit style (you can see other examples in `git log`).

@akien-mga thanks for this suggestion!

clayjohn reviewed (approved):
The content looks great! Thank you

mateuseap commented:
@clayjohn the commit description wasn't looking good so I just adjusted it, sorry for the delay!

![image](https://github.com/godotengine/godot/assets/52019009/b2101a7b-8ef4-4622-af71-5235da11e1fd)

akien-mga commented:
Thanks!

YuriSizov commented:
Cherry-picked for 4.1.3.
